lib/msal-browser/src/interaction_handler/RedirectHandler.ts

```diff
@@ -222,6 +222,9 @@ export class RedirectHandler {
         )) as AuthenticationResult;
 
         this.browserStorage.cleanRequestByState(state);
+        this.browserStorage.removeRequestRetried(
```

Comment:
Does the new request cache entry also get cleaned up here?

Reply:
Good catch. Added removal of redirect request cache item.